script/remove_empty_cached_counts_related_to_overviews.rb

```diff
@@ -4,33 +4,26 @@
 Card::Auth.current_id = Card::WagnBotID
 
 [Card::WikirateAnalysisID, Card::WikirateTopicID, Card::WikirateCompanyID].each do |type|
-  Card.search(
-    right: { codename: 'cached_count'},
-    content: '0', left: { left: {type_id: type}}, return: 'id') do |card_id|
-    puts "~~~\n\nworking on: ~#{card_id}"
-    left_id = Card.select(left_id).where(id: card_id).take
-    begin
-      if (type == Card::WikirateAnalysisID)
-        ana_id = Card.select(:left_id).where(id: left_id).take
-        Card.where(id: ana_id).update_all trash: true
-      end
-      Card.where(id: card_id).update_all trash: true
-      Card.where(id: left_id).update_all trash: true
-    rescue
-      puts "FAILED TO DELETE: #{Card.fetch(card_id).name}".red
-    end
-  end
-end
+  cc_ids = Card.search(
+          right: { codename: 'cached_count'},
+          content: '0', left: { left: {type_id: type}}, return: 'id'
+        )
+  left_ids = Card.select(:left_id).where(id: cc_ids).take
+  ll_ids = Card.select(:left_id).where(id: left_ids).take
+
+
 
-Card.search(right: { codename: 'cached_count'}, content: '0' ).each do |cc|
-  puts "~~~\n\nworking: #{ana.name}"
-  begin
-    cc.update_column :trash, true
-  rescue
-    puts "FAILED TO DELETE: #{cc.name}"
-  end
+  Card.where(type_id: Card::WikirateAnalysisID, id: ll_ids)
+    .update_all trash: true
+  Card.where(id: cc_ids).update_all trash: true
+  Card.where(id: left_ids).update_all trash: true
 end
 
+puts 'handle rest'
+ids = Card.search(
+        right: { codename: 'cached_count'}, content: '0', return: 'id'
```

Comment:
Indent the first parameter one step more than the start of the previous line.<br>Space inside } missing.